apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -105,7 +105,7 @@ public JobsScheduler(
   }
 
   public static void main(String[] args) {
-    log.info("Starting scheduler");
+    log.info("Starting scheduler Updated");
```

Comment:
What's the relevance of "Updated"?

Reply:
This was an unintended change that came in as part of an old commit. Removed in the subsequent ones.